docker-compose.yaml

```diff
@@ -1,5 +1,30 @@
-version: '3'
+version: "3.9"
+
+volumes:
+  mysql-data:
+  prom-data:
+  grafana-data:
+
 services:
+  men-api:
```

Comment:
What is men-api?

Reply:
oh, it should be m`a`n-api, stands from management-api )